server/setup.py

```diff
@@ -58,7 +59,6 @@
         ]
         + (['onnxruntime-gpu>=1.8.0'] if sys.platform != 'darwin' else []),
         'tensorrt': ['nvidia-tensorrt'],
-        'huggingface': ['transformers>=4.16.2'],
```

Comment:
change the dependency key to "mclip"

Reply:
why put mclip in extra requirements, since it is now part of basic models to support

Reply:
I changed my mind, let's use `transformers` as the key. `mclip` actually depends on transformers.